pkg/apis/operatorstatus.openshift.io/v1/types.go

```diff
@@ -25,52 +25,81 @@ type OperatorStatus struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata"`
 
+	// Spec hold the intent of how this operator should behave.
+	Spec OperatorSpec `json:"spec""`
+
```

Comment:
Why prefix the status with type and not spec?

Also there is an extra `"`

Reply:
> Why prefix the status with type and not spec?

`OperatorStatus` conflicted, but sure.